docs: initial instructions (#348)
qlik-oss/nebula.js: docs/README.md

`nebula.js` is a collection of JavaScript libraries and APIs that helps developers integrate visualizations and mashups on top of Qlik's Associative Engine.

It consists of :

- `nucleus`: A product and framework agnostic JavaScript library for building mashups.
- `supernova`: A JavaScript API for consuming and visualizing QIX data.

> - `snapshooter`: A JavaScript API for rendering a `supernova` without a connection to Qlik.

Reviewer: ```suggestion
- `snapshooter`: A JavaScript API for rendering a `supernova` in offline mode.
```
